istioctl/cmd/webhook_test.go

```diff
@@ -1,937 +0,0 @@
-			},
-			shouldFail: true,
-		},
-	}
-
-	for tcName, tc := range testCases {
-		err := tc.opt.Validate()
-		if tc.shouldFail {
-			if err == nil {
-				t.Errorf("%v: opt (%v) should have failed", tcName, tc.opt)
-			}
-		} else if err != nil {
-			t.Errorf("%v: opt (%v) should not fail, but err: %v", tcName, tc.opt, err)
-		}
-	}
-}
-
-func TestDisableWebhookConfig(t *testing.T) {
-	var buf bytes.Buffer
-	testCases := map[string]struct {
-		opt                           disableCliOptions
-		createValidatingWebhookConfig bool
-		createMutatingWebhookConfig   bool
-		shouldFail                    bool
-	}{
-		"disable validating webhook config": {
-			opt: disableCliOptions{
-				disableValidationWebhook:    true,
-				validatingWebhookConfigName: "protovalidate",
-			},
-			createValidatingWebhookConfig: true,
-			createMutatingWebhookConfig:   true,
-			shouldFail:                    false,
-		},
-		"disable mutating webhook config": {
-			opt: disableCliOptions{
-				disableInjectionWebhook:   true,
-				mutatingWebhookConfigName: "protomutate",
-			},
-			createValidatingWebhookConfig: true,
-			createMutatingWebhookConfig:   true,
-			shouldFail:                    false,
-		},
-		"disable validating and mutating webhook configs": {
-			opt: disableCliOptions{
-				disableValidationWebhook:    true,
-				validatingWebhookConfigName: "protovalidate",
-				disableInjectionWebhook:     true,
-				mutatingWebhookConfigName:   "protomutate",
-			},
-			createValidatingWebhookConfig: true,
-			createMutatingWebhookConfig:   true,
-			shouldFail:                    false,
-		},
-		"disable non-existing validating webhook config": {
-			opt: disableCliOptions{
-				disableValidationWebhook:    true,
-				validatingWebhookConfigName: "protovalidate",
-			},
-			createValidatingWebhookConfig: false,
-			createMutatingWebhookConfig:   false,
-			shouldFail:                    true,
-		},
-		"disable non-existing mutating webhook config": {
-			opt: disableCliOptions{
-				disableInjectionWebhook:   true,
-				mutatingWebhookConfigName: "protomutate",
-			},
-			createValidatingWebhookConfig: false,
-			createMutatingWebhookConfig:   false,
-			shouldFail:                    true,
-		},
-	}
-
-	for tcName, tc := range testCases {
-		client := fake.NewSimpleClientset()
-		if tc.createValidatingWebhookConfig {
-			webhookConfig, err := buildValidatingWebhookConfig(
-				[]byte(exampleCACert),
-				"./testdata/webhook/example-validating-webhook-config.yaml")
-			if err != nil {
-				t.Fatalf("%v: err when build validatingwebhookconfiguration: %v", tcName, err)
-			}
-			_, err = createValidatingWebhookConfig(client, webhookConfig, &buf)
-			if err != nil {
-				t.Fatalf("%v: err when creating validatingwebhookconfiguration: %v", tcName, err)
-			}
-		}
-		if tc.createMutatingWebhookConfig {
-			webhookConfig, err := buildMutatingWebhookConfig(
-				[]byte(exampleCACert),
-				"./testdata/webhook/example-mutating-webhook-config.yaml")
```

Comment:
This file "./testdata/webhook/example-mutating-webhook-config.yaml" can be removed too.